src/common/src/log.rs

```diff
@@ -86,7 +89,7 @@ mod tests {
             });
 
             if let Ok(suppressed_count) = RATE_LIMITER.check() {
-                println!("failed to foo bar. suppressed_count = {}", suppressed_count);
+                println!("failed to foo bar. suppressed_count = {:?}", suppressed_count);
```

Comment:
What does the `{:?}` look like?

Reply:
It does not look good. 😄 We usually record it as a tracing field in production code. Updated the test code accordingly and it now looks like this:

![Uploading image.png…]()